otsdaq-mu2e/ROCCore/ROCCoreVInterface.cc

```diff
@@ -133,6 +174,20 @@ void ROCCoreVInterface::writeBlock(const std::vector<DTCLib::roc_data_t>& writeD
 
 }  // end readBlock()
 
+//==================================================================================================
+void ROCCoreVInterface::GetStatus(__ARGS__)
+{
```

Comment:
I think we should just make these functions 'pure virtual' which means in the .h they are declared = 0. Then all inheriting classes must define. No need for code in the .cc